contracts/Pricer.sol

```diff
@@ -1,3 +1,4 @@
+/* solhint-disable-next-line compiler-fixed */
```

Comment:
on L34: Pricer is not PricerInterface; the dependency direction is reversed; pricer calls on contracts, for now other contracts dont call on pricer; neither is there anything general about pricer worthy of generalising in an interface

Reply:
as we concluded we will keep PricerInterface.sol